docker/run_first.sh

```diff
@@ -29,6 +29,9 @@ DOCKER_IP="localhost"
 # clean migration files in every environment
 rm -f ../fuel/app/config/**/migrations.php
 
+# generate a self-signed ssl cert
+openssl req -subj '/CN=localhost' -x509 -newkey rsa:4096 -nodes -keyout ./config/nginx/key.pem -out ./config/nginx/cert.pem -days 365
```

Comment:
Are there additional requirements to getting this to run successfully? It's failing for me:
```
writing new private key to './config/nginx/key.pem'
./config/nginx/key.pem: Is a directory
```

Reply:
Scratch that, removing `key.pem` and `cert.pem` and rerunning the command above worked. Chalk up another one to "didn't start from scratch and got into a misconfiguration"

Reply:
Hmm, strange.  Not sure what was going on there.